Makefile.PL

```diff
@@ -125,9 +133,9 @@ WriteMakefile(
       resources =>
       {
         license     => 'http://dev.perl.org/licenses/',
```

Comment:
The license resource is an array in meta-spec 2.

Reply:
fixed